### ISSUE TITLE
Firefox iOS is missing page zoom

### ISSUE THREAD

**Issue by RespectIsEverything** [mozilla-mobile/firefox-ios#9012]
**## Steps to reproduce**
View page rendered in Safari using page zoom settings.
View same page in Firefox iOS

**### Expected behavior**
Whatever implementation Safari is using to render would also be available in Firefox iOS as it uses Webkit to render.

**### Actual behavior**
Feature is missing from Firefox

**### Device & build information**
* Device: iPhone 12 Mini
* Build version: 36.0

**### Notes**
Pictures included showing the difference in rendering.

![Rendering differences](https://user-images.githubusercontent.com/55006522/130398159-a4cb67db-8ee5-4c2e-9ccd-941ab2d1da68.png)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-3093)

**Comment by RespectIsEverything** [comment]
Just saw this is a duplicate of 7596 and 8230.
A pull request was submitted in March 2021.

**Comment by ORIOLESFan02** [comment]
Can confirm. It’s pretty annoying especially when you have something on there that won’t zoom out regardless of what you do. For example. I can’t see my class schedule on my iPhone cause I can’t zoom out.
